dogechain-lab/dogechain
# Implement eth_getFilterLogs

# Description

This PR implement `eth_getFilterLogs` JSON-RPC method, which is missing from `dogechain` JSON-RPC api.

The PR is merging from 0xPolygon Edge [PR 538](https://github.com/0xPolygon/polygon-edge/pull/538).

# Changes include


# Checklist


## Testing


### Manual tests

* Deploy a dev mode server
* Deploy a simple contract with two topic logs.
* Call `eth_newFilter` to get a filter id.
* Call contract for some logs.
* Call `eth_getFilterLogs` to get one topic logs.

We should be able to filter out the logs. 

# Documentation update

https://github.com/dogechain-lab/jury-docs/pull/1
